recipes-multimedia/omx-module/omx-user-module.bb

```diff
@@ -0,0 +1,165 @@
+DESCRIPTION = "OMX Media Components RZG2"
+LICENSE = "CLOSED"
+require include/rzg2l-sbc-modules-common.inc
+require omx-package-rzg2l.inc
+
+DEPENDS = " \
+    kernel-module-mmngr mmngr-user-module \
+    vspmif-user-module kernel-module-vspmif \
+    kernel-module-vspm \
+"
+
+# Task Control. Compile is not performed when not installing OMX Video and Audio Libs.
+# Note) dummy-omx-user-module.inc does not exist.
+INCLUDE_FILE = '${@oe.utils.conditional("USE_OMX_COMMON", "1", "dummy", "deltask", d )}'
+include ${INCLUDE_FILE}-omx-user-module.inc
+
+DEPENDS += '${@oe.utils.conditional("USE_VIDEO_OMX", "1", "kernel-module-uvcs-drv", "", d )}'
+
+inherit autotools
+
+includedir = "${RENESAS_DATADIR}/include"
+CFLAGS += " -I${STAGING_DIR_HOST}${RENESAS_DATADIR}/include"
+PACKAGE_ARCH = "${MACHINE_ARCH}"
+
+OMX_EVA_PREFIX = '${@oe.utils.conditional("USE_OMX_EVA_PKG", "1", "EVA", "", d )}'
+
+SRC_URI:rzg2l-sbc = " \
+	file://${OMX_PKG_TAR} "
+
+# SRC directory name
+OMX_COMMON_SRC:rzg2l-sbc = '${@oe.utils.conditional("USE_OMX_COMMON", "1", "${OMX_EVA_PREFIX}omx_common_lib_package", "", d )}'
+OMX_VIDEO_DEC_COMMON_SRC:rzg2l-sbc = '${@oe.utils.conditional("USE_VIDEO_DEC", "1", "${OMX_EVA_PREFIX}video_dec_common_package", "", d )}'
+OMX_VIDEO_ENC_COMMON_SRC:rzg2l-sbc = '${@oe.utils.conditional("USE_VIDEO_ENC", "1", "${OMX_EVA_PREFIX}video_enc_common_package", "", d )}'
+OMX_H264_DEC_SRC:rzg2l-sbc = '${@oe.utils.conditional("USE_H264D_OMX", "1", "${OMX_EVA_PREFIX}h264_decoder_package", "", d )}'
+OMX_H264_ENC_SRC:rzg2l-sbc = '${@oe.utils.conditional("USE_H264E_OMX", "1", "${OMX_EVA_PREFIX}h264_encoder_package", "", d )}'
+
+
+OMX_VIDEO_SRC_LIST = " \
+    ${OMX_COMMON_SRC} \
+    ${OMX_VIDEO_DEC_COMMON_SRC} \
+    ${OMX_VIDEO_ENC_COMMON_SRC} \
+    ${OMX_H264_DEC_SRC} \
+    ${OMX_H264_ENC_SRC} \
+"
+S:rzg2l-sbc = "${WORKDIR}/${OMX_PKG}"
+
+do_fetch[file-checksums] = ""
+do_compile[noexec] = "1"
+do_package_qa[noexec] = "1"
```

Comment:
Please add a comment explaining that this recipe is using pre-built closed-source binaries, which is why the do_fetch, do_compile, and other related tasks are skipped.